articles/tutorials/index.md

```diff
@@ -0,0 +1,10 @@
+# Tutorials
+
+We try to help you understand Intent Architect as a newcomer by providing a couple of tutorials to work through.
+Please choose the one that closely relates to the feature you're trying to learn.
+
+### [Getting Started - Create an ASP.NET Core web app](create_an_aspnetcore_web_app.md)
+Let us show you how easy it is to create a new ASP.NET Core Web Application using Intent Architect and have it up and running in 5 minutes.
+
+### [Getting Started - Create your own Module](create_your_own_module.md)
```

Comment:
Let's change this to an xref with a blank titles so it will automatically populate the title from the article.